cmake/projects/Lua/hunter.cmake

```diff
@@ -0,0 +1,23 @@
+if(DEFINED HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_LUA_HUNTER_CMAKE_ 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Lua
+    VERSION
+    "5.3.2"
+    URL
+    "https://github.com/hunter-packages/lua/archive/5.3.2.tar.gz"
```

Comment:
I've applied some fixes for Linux:
* https://github.com/hunter-packages/lua/commit/54d84610f3a2fda8b0afdfc45d2977c27d788ea9
* https://github.com/hunter-packages/lua/commit/1054f9fa7a86dd15ecdc085430c038883bbdf0ae

Please update archive: https://github.com/hunter-packages/lua/releases/tag/v5.3.2-p0